File: backend/Modulos/models/estructuraCentroCostoUsuario.go
package models

// EstructuraCentroCostoUsuario is...
type EstructuraCentroCostoUsuario struct {
	SERVIDOR                    string               `json:"servidor"`
	IDUSUARIO                   int                  `json:"idusuario"`
	IDCENTROCOSTO               int                  `json:"idcentrocosto"`
	HDGCODIGO                   int                  `json:"hdgcodigo"`
	ESACODIGO                   int                  `json:"esacodigo"`
	CMECODIGO                   int                  `json:"cmecodigo"`
	GLOUNIDADESORGANIZACIONALES string               `json:"glounidadesorganizacionales"`
	DETALLE                     []CentroCostoUsuario `json:"detalle"`
	ACCION                      string               `json:"accion"`
}

File: frontend/src/app/models/entity/estructura-centro-costo-usuario.ts
import { CentroCostoUsuario } from './centro-costo-usuario';

export class EstructuraCentroCostoUsuario {
        constructor (
             public servidor          ?: string,
             public idusuario         ?: number,
             public idcentrocosto     ?: number,
             public hdgcodigo         ?: number,
             public esacodigo         ?: number,
             public cmecodigo         ?: number,
             public glounidadesorganizacionales   ?: string,
             public accion            ?: string,
             public detalle           ?:CentroCostoUsuario[],
         ) {}
    }
